src/models/Scholarships.test.ts

```diff
@@ -2,13 +2,48 @@ import { firestore } from 'firebase';
 import { clearFirestoreData, initializeTestApp } from '../lib/testing';
 import AmountType from '../types/AmountType';
 import ScholarshipAmount from '../types/ScholarshipAmount';
-import { converter } from './Scholarships';
+import Scholarships, { converter } from './Scholarships';
 
 const app = initializeTestApp({ projectId: 'scholarship-test' });
 
-beforeEach(async () =>
-  clearFirestoreData(app.options as { projectId: string })
-);
+// Creates a scholarship to be stored with the given amount.
+// The deadline is set to the time this function is called + index ms.
+function create(amount: ScholarshipAmount, index: number) {
+  return Scholarships.new({
+    name: amount.toString(),
+    amount,
+    deadline: new Date(new Date().getTime() + index),
+    website: 'foo.com',
+    description: 'some description',
+  });
+}
+
+// All scholarships, sorted by deadline
```

Comment:
The ordering is guaranteed by the last parameter. Each scholarship will be created at the same time or right after the previous scholarship but the index increases so in the end this ends up being the sorted by deadline order from earliest to latest.